listenbrainz/db/stats.py

```diff
@@ -43,16 +43,16 @@ def insert_user_stats(user_id, artists, recordings, releases, artist_count):
 
     artist_stats = {
         'count': artist_count,
-        'all_time': artists,
+        'prev_month': artists,
     }
 
     recording_stats = {
-        'all_time': recordings,
+        'prev_month': recordings,
```

Comment:
We should prolly just specify the month here in the key, like `2019-01` so that we know exactly which month the data is for. With prev month, we might have data for different months for different users.